YouTubePlayer/YouTubePlayer/YouTubePlayer.swift

```diff
@@ -181,8 +189,12 @@ public class YouTubePlayerView: UIView, UIWebViewDelegate {
     }
 
     private func evaluatePlayerCommand(command: String) {
+        queryPlayerCommand(command)
+    }
+    
+    private func queryPlayerCommand(command: String) -> String? {
```

Comment:
I can do that, it seemed easier to reason about and more explicit to have one place that ignores the return value.

Commit incoming